From a91913b496a07d6e7edb13a48c66a22e642afb9b Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <ignatenko@redhat.com>
Date: Thu, 9 Jun 2016 17:15:23 +0200
Subject: [PATCH 2/2] test/runner: add support for getting executable from ENV

Signed-off-by: Igor Gnatenko <ignatenko@redhat.com>
---
 CMakeLists.txt | 1 +
 test/runner.py | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 55a66cf..8e864cb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,6 +82,7 @@ if (ENABLE_TESTS)
   if (PYTHONINTERP_FOUND)
     add_test (NAME test-py
               COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test/runner.py)
+    set_tests_properties(test-py PROPERTIES ENVIRONMENT HOEDOWN=./hoedown)
   endif ()
   if (PERL_FOUND)
     add_test (NAME test-pl
diff --git a/test/runner.py b/test/runner.py
index 7cbb27d..687ef4f 100755
--- a/test/runner.py
+++ b/test/runner.py
@@ -10,7 +10,7 @@ import unittest
 
 TEST_ROOT = os.path.dirname(__file__)
 PROJECT_ROOT = os.path.dirname(TEST_ROOT)
-HOEDOWN = [os.path.abspath(os.path.join(PROJECT_ROOT, 'hoedown'))]
+HOEDOWN = [os.environ.get('HOEDOWN', os.path.abspath(os.path.join(PROJECT_ROOT, 'hoedown')))]
 TIDY = ['tidy', '--show-body-only', '1', '--show-warnings', '0',
         '--quiet', '1']
 CONFIG_PATH = os.path.join(TEST_ROOT, 'config.json')
-- 
2.7.4

